### PR TITLE
Format with prettier

### PR DESCRIPTION
**Pull request by D4nte**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -20,6 +20,18 @@ pipeline {
   }
 
   stages {
+    stage('Deps') {
+      steps {
+        sh "yarn install --frozen-lockfile"
+      }
+    }
+
+    stage('Checks') {
```
Comment (D4nte) [inline Jenkinsfile]: @jakubgs how do I add this step as part of a pull request check please?

Comment (jakubgs) [reply Jenkinsfile]: One way would be through GitHub Actions. We could do it in Jenkins, but that would require a separate `Jenkinsfile` that runs just the checks and updates github PR with check results.

**Review by jakubgs — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (D4nte) [reply Jenkinsfile]: I wasn't too sure if we want to mix GitHub actions and Jenkins for CI jobs but it makes sense to use GH actions as it is a quick/low resource job. Thanks!

Comment (D4nte) [reply Jenkinsfile]: I'll setup the GH action as part of this PR.

Comment (D4nte) [reply Jenkinsfile]: @jakubgs can you please enable github action on this repo or provide me with admin rights so I can do it? Cheers.

**Review by jakubgs — APPROVED** [review]
Enabled.

**Comment by D4nte** [comment]
> Enabled.

Thanks! I am getting an error: https://github.com/vacp2p/docs.wakuconnect.dev/actions/runs/1826574789

First

```
actions/checkout@v2.3.3 and actions/setup-node@v2 are not allowed to be used in vacp2p/docs.wakuconnect.dev. Actions in this workflow must be: within a repository owned by vacp2p.
```

According to https://github.community/t/checkout-v2-error-actions-in-this-workflow-must-within-a-repository-owned-by-organization/136315 it is because the "allow local action only" checkbox is ticked.

**Comment by jakubgs** [comment]
Changed.

**Pull request merged**